<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TGo4MarkerSettings</class>
 <widget class="QDialog" name="TGo4MarkerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global marker label setup:</string>
  </property>
  <property name="windowIcon">
   <iconset resource="go4icons.qrc">
    <normaloff>:/icons/go4logo2_big.png</normaloff>:/icons/go4logo2_big.png</iconset>
  </property>
  <property name="toolTip">
   <string>Check properties for the markers of regions (conditions), and points. Changes are applied for newly created markers only!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="ConditionGroupBox">
     <property name="title">
      <string>Windows/Polygons</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="DrawLabelButton">
        <property name="text">
         <string>Draw region label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="DrawLimButton">
        <property name="text">
         <string>Display region limits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="DrawIntButton">
        <property name="text">
         <string>Display Integral</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="DrawCountsButton">
        <property name="text">
         <string>Display Counts maximum</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="DrawXmeanButton">
        <property name="text">
         <string>Display X mean</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="DrawYmeanButton">
        <property name="text">
         <string>Display Y mean</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="DrawXrmsButton">
        <property name="text">
         <string>Display X rms</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="DrawYrmsButton">
        <property name="text">
         <string>Display Y rms</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="DrawXmaxButton">
        <property name="text">
         <string>Display X maximum</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="DrawYmaxButton">
        <property name="text">
         <string>Display Y maximum</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="WLabelFormatEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>%.4E</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="WLabelFormatLabel">
          <property name="text">
           <string>Number Format</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="MarkerGroupBox">
     <property name="title">
      <string>Point markers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="DrawPLabelButton">
        <property name="text">
         <string>Draw marker label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="DrawPConnectorButton">
        <property name="text">
         <string>Draw label connection line</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="DrawPXButton">
        <property name="text">
         <string>Display X coord.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="DrawPYButton">
        <property name="text">
         <string>Display Y coord.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="DrawPXbinButton">
        <property name="text">
         <string>Display X bin#</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="DrawPYbinButton">
        <property name="text">
         <string>Display Y bin#.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="DrawPCountsButton">
        <property name="text">
         <string>Display bin contents</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="PLabelFormatEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>%.4E</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PLabelFormatLabel">
          <property name="text">
           <string>Number Format</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>210</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Take settings.</string>
       </property>
       <property name="icon">
        <iconset resource="go4icons.qrc">
         <normaloff>:/icons/OK16.png</normaloff>:/icons/OK16.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Close panel without changes.</string>
       </property>
       <property name="icon">
        <iconset resource="go4icons.qrc">
         <normaloff>:/icons/close16.png</normaloff>:/icons/close16.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>DrawLabelButton</tabstop>
  <tabstop>DrawLimButton</tabstop>
  <tabstop>DrawIntButton</tabstop>
  <tabstop>DrawCountsButton</tabstop>
  <tabstop>DrawXmeanButton</tabstop>
  <tabstop>DrawYmeanButton</tabstop>
  <tabstop>DrawXrmsButton</tabstop>
  <tabstop>DrawYrmsButton</tabstop>
  <tabstop>DrawXmaxButton</tabstop>
  <tabstop>DrawYmaxButton</tabstop>
  <tabstop>WLabelFormatEdit</tabstop>
  <tabstop>DrawPLabelButton</tabstop>
  <tabstop>DrawPConnectorButton</tabstop>
  <tabstop>DrawPXButton</tabstop>
  <tabstop>DrawPYButton</tabstop>
  <tabstop>DrawPXbinButton</tabstop>
  <tabstop>DrawPYbinButton</tabstop>
  <tabstop>DrawPCountsButton</tabstop>
  <tabstop>PLabelFormatEdit</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="go4icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>TGo4MarkerSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>TGo4MarkerSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>TGo4MarkerSettings</receiver>
   <slot>setFlags()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
